classes/class-p4-post-campaign.php

```diff
@@ -427,10 +428,11 @@ public static function get_body_font( $meta ): array {
 					'plastic'   => 'Montserrat',
 				];
 				$theme              = self::get_theme( $meta );
-				$body_font          = $campaigns_font_map[ $theme ];
+
+				$css_vars['campaign_body_font'] = $campaigns_font_map[ $theme ];
 			}
 
-			return [ 'campaign_body_font' => $body_font ];
+			return $css_vars;
```

Comment:
If the `$css_vars` is passed by reference then this function doesn't need to return it as it's not used. Though personally I prefer not using passing by reference, as in function calls you can't really see that that is happening, and assigning the return value makes it a bit more obvious what is happening, though the extra assignment does make the syntax a bit more verbose.

Reply:
Good point, I'll revert to the assignments.